BlueprintUI/Sources/Layout/Builder/EqualStackChild.swift

```diff
@@ -0,0 +1,17 @@
+/// Wraps around an Element for consistency with `StackChild` and `GridRowChild`.
+/// In the future this struct could hold traits used for laying out inside an EqualStack
+extension EqualStack {
+    public struct Child {
```

Comment:
Can you move this file's contents into `EqualStack.swift`? It's very much part of that implementation, I think.

Reply:
Bump here

Reply:
Done